neqo-transport/src/fc.rs

```diff
@@ -239,6 +239,11 @@ where
         self.max_data = new_max;
         self.frame_pending = false;
     }
+
+    pub fn change_max_stream_data(&mut self, max_data: u64) {
```

Comment:
This needs to be renamed to something about active, like set_active(max).  Then we can use it for other things.